### ISSUE TITLE
Docs to help you

### ISSUE THREAD

**Issue by 0Franky** [InoriRus/Kyty#96]
Hi @InoriRus,
first of all thank you for your contribution.

I started looking at the project and it seems very big and not easy to understand.
I wanted to ask you if it's possible to create a roadmap/tasks and related documentation of what to do for that specific task.
For example I read that the save directories and language are hardcoded. 
If I would like to contribute, where can I find the relevant code? I only found some xml that contain references on the language.

It would also be useful to perhaps create a discord channel where talk to each other more easily and perhaps give support.

Thanks in advance!

**Comment by AboMedoz** [comment]
Bro the project is abandoned 💀

**Comment by 0Franky** [comment]
@MedoGamer888 
Yeah, but the community is amazing. maybe there are those who are interested (like me) and would like to contribute. many small contributions have allowed the development of many apps that are now huge. if it's not a problem for the original developer to at least give an explanation of the project, maybe it can come back to life and maybe one day be able to have a decent product.

**Comment by AboMedoz** [comment]
The developer has abandoned the project and there's no way to reach inorirus just move on bro
